01-CodeReviewEx/Shop-Ex/Program.cs

```diff
@@ -12,9 +12,33 @@ static void Main(string[] args)
         {
             
          
+         Account[] myAccounts = new Account[10];
+            Account a = new Account();
```

Comment:
If you're setting anyway all of the parameters, please provide a constructor for it

Reply:
btw. why do we need list of size 10 and insert only one account into it?

Reply:
I am not sure how to do it in c#, but in Java I would propose to do calculateFee(List.of(a));😎